src/sunmemory/cuda/sundials_cuda_memory.cu

```diff
@@ -383,6 +394,6 @@ int SUNMemoryHelper_GetAllocStats_Cuda(SUNMemoryHelper helper,
     *bytes_allocated      = SUNHELPER_CONTENT(helper)->bytes_allocated_uvm;
     *bytes_high_watermark = SUNHELPER_CONTENT(helper)->bytes_high_watermark_uvm;
   }
-  else { return -1; }
-  return 0;
+  else { return SUN_ERR_ARG_OUTOFRANGE; }
```

Comment:
Since `mem_type` is an enumerated type, shouldn't this be
```suggestion
  else { return SUN_ERR_ARG_INCOMPATIBLE; }
```

Reply:
https://en.cppreference.com/w/cpp/language/enum#:~:text=An%20enumeration%20is%20a%20distinct,constants%20(%22enumerators%22).